networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -139,13 +139,104 @@ include::modules/nw-multus-tap-object.adoc[leveloffset=+2]
+
+You can create an `NodeNetworkConfigurationPolicy` object, part of the `nmstate.io/v1` API group, to declaratively create the mapping. This API is provided by the NMState Operator. By using this API, you can apply the bridge mapping to nodes that match the `nodeSelector` that you specify, such as `node-role.kubernetes.io/worker: ''` to apply to all worker nodes in your cluster.
+
+When attaching an additional network, you can either use the existing `br-ex` bridge or create a new bridge. Which approach to use depends on your specific network infrastructure.
+
+- If your nodes include only a single network interface, you must use the existing bridge. This network interface is owned and managed by OVN-Kubernetes and you must not remove it from the `br-ex` bridge or alter the interface configuration, or your cluster network will stop working correctly.
+- If your nodes include several network interfaces, you attach a different network interface to a new bridge, and use that for your additional network. This approach provides for traffic isolation from your primary cluster network.
+
+In the following example, the `physnet1` additional network is attached to the `ovsbr1`. Through this attachment, the additional network is available to the OVN-Kubernetes network plugin as an additional network.
+
+.Example mapping for sharing a bridge
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: mapping <1>
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' <2>
+  desiredState:
+    ovn:
+      bridge-mappings:
+      - localnet: physnet1 <3>
+        bridge: br-ex <4>
+        state: present <5>
+----
+<1> The name for the configuration object.
+<2> A node selector that specifies the nodes to apply the node network configuration policy to.
+<3> The name for the network interface.
+<4> The name of the bridge on the node. This value is required only if you specify `state: present`.
+<5> The state for the mapping. Must be either `present` to add the bridge or `absent` to remove the bridge. The default value is `present`.
+
+// TODO - fix
+In the following example, the `physnet1` network interface is attached to the `ovsbr1` bridge. Through this attachment, the network interface is available to the OVN-Kubernetes network plugin as an additional network.
+
+.Example mapping for nodes with multiple interfaces
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: ovs-br1-multiple-networks
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' <1>
+spec:
+  desiredState:
+    interfaces:
+    - name: ovsbr1
+      description: Dedicated OVS bridge with eth1 as a port allowing all VLANs and untagged traffic through
+      type: ovs-bridge
+      state: up
+      bridge:
+        options:
+          stp: true
+        port:
+        - name: eth1 <2> # <name of the node interface>
+    ovn:
+      bridge-mappings:
+      - localnet: physnet2 <3>
```

Comment:
we probably should highlight this must match the name of the network in the NAD (nad.spec.config.name).